glossary/other/model-of-quantum-computation.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Model of Quantum Computation
+parent: Other
+nav_order: 2
```

Comment:
This should be 1 and the other files bumped up?

Reply:
You're right. Sorry, forgot about that!